src/Android/Xamarin.Android/Samples/Symbology/CustomDictionaryStyle/readme.md

```diff
@@ -0,0 +1,40 @@
+# Custom dictionary style
+
+Use a custom dictionary style (.stylx) to symbolize features using a variety of attribute values.
+
+![Custom dictionary style](CustomDictionaryStyle.jpg)
+
+## Use case
+
+When symbolizing geoelements in your map, you may need to convey several pieces of information with a single symbol. You could try to symbolize such data using a unique value renderer, but as the number of fields and values increases, that approach becomes impractical. With a dictionary renderer you can build each symbol on-the-fly driven one or more several attribute values and handle a nearly infinite number of unique combinations.
+
+## How to use the sample
+
+Pan and zoom the map to see the symbology from the custom dictionary style.
+
+## How it works
+
+1. Create a new `DictionarySymbolStyle` by passing in the path to the custom style (.stylx).
+2. Define symbol attribute overrides (if any) using a collection of key-value pairs: `configured-attribute-name : override-attribute-name`.
+3. Define text attribute overrides (if any) using a collection of key-value pairs: `configured-attribute-name : override-attribute-name`.
+4. If necessary, provide new values for configuration settings defined in `DictionarySymbolStyle.getConfigurations()`.
```

Comment:
Don't forget to adjust the code snippets to match .NET style (properties instead of getters/setters)